KlubJagiellonski/pola-web
# PW-32: api errors handling

Steps to test:
1. Type any text to search input.
2. List of mocked products should be visible.
3. Each product on the list is clickable and open modal with product details.
